servicebus/data-plane/azure-servicebus/pom.xml

```diff
@@ -1,100 +1,99 @@
-					<showWarnings>true</showWarnings>
-				</configuration>
-			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-	        	<artifactId>maven-surefire-plugin</artifactId>
-	        	<version>2.20</version>
-	        	<configuration>
-	        		<forkCount>0</forkCount> <!-- Speeds up test execution using multiple JVMs. Set it to 0 if you want to run test sequentially in the build vm itself -->
-	        		<reuseForks>true</reuseForks>
-	        	</configuration>
-	      	</plugin>
-		</plugins>
-	</build>
+  <parent>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-servicebus-parent</artifactId>
+    <version>2.0.0</version><!-- Need not change for every release -->
+    <relativePath>../pom.xml</relativePath>
+  </parent>
 
-	 <dependencies>
-	 	<dependency>
-	 	    <groupId>org.apache.qpid</groupId>
-	 	    <artifactId>proton-j</artifactId>
-	 	    <version>${proton-j-version}</version>
-	 	</dependency>
-	  	<dependency>
-			 <groupId>com.microsoft.azure</groupId>
-			 <artifactId>qpid-proton-j-extensions</artifactId>
-			 <version>1.1.0</version>
-		 </dependency>
-	 	<dependency>
-			 <groupId>com.microsoft.azure</groupId>
-			 <artifactId>adal4j</artifactId>
-			 <version>1.3.0</version>
-		 </dependency>
-		<dependency>
-	 	    <groupId>junit</groupId>
-	 	    <artifactId>junit</artifactId>
-	 	    <version>${junit-version}</version>
-	 	    <scope>test</scope>
-		</dependency>
-		<dependency>
-	 	    <groupId>org.slf4j</groupId> 
-	 	    <artifactId>slf4j-api</artifactId> 
-	 	    <version>${slf4j-version}</version>
-	  	</dependency>
-		<!--<dependency>
-	 	    <groupId>org.slf4j</groupId> 
-	 	    <artifactId>slf4j-jdk14</artifactId>
-	 	    <version>${slf4j-version}</version>
-	  	</dependency>-->
-	  	<dependency>
-	 	    <groupId>org.slf4j</groupId> 
-	 	    <artifactId>slf4j-log4j12</artifactId> 
-	 	    <version>${slf4j-version}</version>
-	  	</dependency>
-	  	<!--<dependency>
-	 	    <groupId>org.slf4j</groupId> 
-	 	    <artifactId>slf4j-log4j12</artifactId> 
-	 	    <version>${slf4j-version}</version>
-	  	</dependency>-->
-		 <dependency>
-			 <groupId>org.asynchttpclient</groupId>
-			 <artifactId>async-http-client</artifactId>
-			 <version>2.5.2</version>
-		 </dependency>
-	</dependencies>
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+  </properties>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I don't think we need this plugin here because it should be defined in the parent. IIRC, we also removed this from event-hub's pom.xml

Reply:
deleted